.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: CI
+
+on:
+  push:
+    branches: ["master"]
+  pull_request:
+    branches: ["master"]
+
+jobs:
+  tox:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 5
+      matrix:
+        python-version:
+        - 2.7
+        - 3.6
+        - 3.7
+        - 3.8
+        - pypy
+        - pypy3
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install tox
+      run: |
+        python -m pip install --upgrade pip setuptools
+        pip install --upgrade tox tox-gh-actions
+    - name: Initialize tox envs
+      run: |
+        tox --parallel auto --notest
+    - name: Test with tox
+      run: |
+        tox --parallel 0
```

Comment:
FYI, instead of disabling parallelism, it's more responsive to use `TOX_PARALLEL_NO_SPINNER=1` (e.g. https://github.com/ansible/pylibssh/blob/2a3a4b2/.github/workflows/build-test-n-publish.yml#L170) and `--parallel-live` (e.g. https://github.com/ansible/pylibssh/blob/2a3a4b2/.github/workflows/build-test-n-publish.yml#L729)